PowerEditor/installer/nativeLang/german.xml

```diff
@@ -240,6 +248,9 @@
 					<Item id="44094" name="&amp;9. Tab"/>
 					<Item id="44095" name="&amp;Nächster Tab"/>
 					<Item id="44096" name="&amp;Vorheriger Tab"/>
+					<Item id="44097" name="Überwachen (Ende -f)"/>
```

Comment:
You should not translate a well known command. <http://linux.die.net/man/1/tail>